.github/mergeable.yml

```diff
@@ -32,6 +32,9 @@ mergeable:
       - do: label
         must_exclude:
           regex: 'wip'
+      - do: label
+        must_exclude:
+          regex: 'dont-merge'
```

Comment:
This won't be any different than "wip"

Reply:
It's is okay.  I wanted to avoid sitation similar to https://github.com/apache/airflow/pull/7510#issuecomment-591363625

Reply:
We can avoid that by adding "[WIP]" in the title too

Reply:
I've finished the performance changes, but I want them not merged so that more people can review. Changes that are WIP are a different kind of change than DONT-MERGE.

Reply:
Fair Enough